core/app/crash/reporting/reporting.go

```diff
@@ -81,58 +84,66 @@ func Disable() {
 }
 
 // ReportMinidump encodes and sends a minidump report to the crashURL endpoint.
-func ReportMinidump(r Reporter, minidumpName string, minidumpData []byte) error {
+func ReportMinidump(r Reporter, minidumpName string, minidumpData []byte) (string, error) {
 	if disable != nil {
-		if err := r.reportMinidump(minidumpName, minidumpData, crashURL); err != nil {
-			return err
+		if res, err := r.reportMinidump(minidumpName, minidumpData, crashURL); err != nil {
```

Comment:
assuming `reportMinidump` returns only a value or error for all codepaths, you can just:

```go
return r.reportMinidump(minidumpName, minidumpData, crashURL)
```